app/ts/Elicitation/Interface/IExactSwingRatio.ts

```diff
@@ -0,0 +1,5 @@
+export default interface IExactSwingRatio {
+  type: 'exact swing';
+  criteria: [string, string];
+  ratio: number;
+}
```

Comment:
I think it is a good idea to add a 'elicitationMethod' property to these, it will make our lives a lot easier